README.md

```diff
@@ -8,85 +8,45 @@ The Teleporter protocol, on the other hand, is implemented at the smart contract
 
 - [Overview](#overview)
 - [Setup](#setup)
-  - [Docker Setup](#docker-setup)
-  - [General Setup](#general-setup)
+  - [Initialize the repository](#initialize-the-repository)
+  - [Dependencies](#dependencies)
 - [Structure](#structure)
-- [Run the Docker integration tests](#run-the-docker-integration-tests)
+- [Run a local testnet in Docker](#run-a-local-testnet-in-docker)
   - [Start up the local testnet](#start-up-the-local-testnet)
-  - [Run the integration tests in Docker containers](#run-the-integration-tests-in-docker-containers)
   - [Additional notes](#additional-notes)
-  - [Run tests on Fuji Testnet](#run-tests-on-fuji-testnet)
 - [E2E tests](#e2e-tests)
+  - [Run the E2E tests on another network](#run-the-e2e-tests-on-another-network)
 - [ABI Bindings](#abi-bindings)
 - [Docs](#docs)
 - [Resources](#resources)
 
 ## Setup
+### Initialize the repository
+- Get all submodules: `git submodule update --init --recursive`
 
-### Docker Setup
-
-- Install Docker:
-
-```
-sudo apt-get install \
-    ca-certificates \
-    curl \
-    gnupg \
-    lsb-release
-curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
-echo \
-  "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu \
-  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
-sudo apt-get update
-sudo apt-get install docker-ce docker-ce-cli containerd.io
-sudo systemctl start docker # Docker will start automatically on future system boots
-```
```

Comment:
Do we? My understanding is that Ginkgo installs everything needed to run the E2E tests in its own package, no need to install anything else.

In any case, I removed this section because I felt it was overly detailed when compared to the rest of the repository documentation. There is any number of resources for installing Docker, and there's nothing specific about our setup that warrants separate documentation.

Reply:
Yeah I guess you're right! How convenient 😄